cdm/src/main/java/ucar/nc2/Group.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 1998-2017 John Caron and University Corporation for Atmospheric Research/Unidata
```

Comment:
I would leave John in this copyright line since he made changes after he left. I don't think it matters too much for contributors who have not added their name to the copyright explicitly, as they still keep their copyright regardless, but I don't think we should remove it if it is explicitly added. Just my 2c.

Reply:
I disagre. As far as we know John was still an employee when those changes were made.

Reply:
Ok, that makes sense then.